ipfs/config.go

```diff
@@ -13,7 +13,7 @@ type Config struct {
 	ServeAPI bool   `mapstructure:"serve-api"`
 }
 
-// DefaultConfig returns a default config different from the default IPFS config.
+// DefaultFullNodeConfig returns a default config different from the default IPFS config.
```

Comment:
I would not rename this unless we rename the function too (which is orthogonal to this or anyways).

Reply:
I think this an artifact of Goland.Refactor of some uncommitted work

Reply:
So I'll remove that